src/lint/practice_exercises.nim

```diff
@@ -0,0 +1,64 @@
+import std/[json, os]
+import ".."/helpers
+import "."/validators
+
+proc isValidAuthorOrContributor(data: JsonNode, context: string, path: string): bool =
+  if isObject(data, context, path):
+    result = true
+    if not checkString(data, "github_username", path):
+      result = false
+    if not checkString(data, "exercism_username", path, isRequired = false):
+      result = false
+
+proc checkFiles(data: JsonNode, context, path: string): bool =
+  result = true
+  if hasObject(data, context, path):
+    if not checkArrayOfStrings(data, context, "solution", path):
+      result = false
+    if not checkArrayOfStrings(data, context, "test", path):
+      result = false
+    if not checkArrayOfStrings(data, context, "example", path):
+      result = false
+  else:
+    result = false
+
+proc isValidPracticeExerciseConfig(data: JsonNode, path: string): bool =
```

Comment:
Once again, this logic is almost identical to the concept exercises' logic. For now, the only difference is that there is no `forked_from` property for practice exercises. There will be more difference later when more practice exercise checks are implemented. We could consider extracting the shared logic to a helper.